source/user-manual/wazuh-dashboard/config-file.rst

```diff
@@ -724,6 +724,105 @@ Set the footer of the PDF reports. To use an empty footer, type a space " " in t
 | **Allowed values** | Any string                 |
 +--------------------+----------------------------+
 
+ISM rollover policy
+-------------------
+
+.. warning::
+
+   Modifying these options has no effect if you do it once the Wazuh dashboard started for the first time. Read about different methods to customize the rollover policy in :doc:`/user-manual/wazuh-indexer/index-life-management`.
+
+ism.rollover.enabled
+^^^^^^^^^^^^^^^^^^^^
+
+Toggles the verification of the rollover policy. If set to ``true``, the rollover policy will be applied to the indices that match the index patterns defined in ``ism.rollover.index_patterns``, only if there isn't any other policy managing these indices.
+
+This option does not enable or disable the policy itself. 
```

Comment:
If not then what does? How to enable and disable? The option name is "enabled", what does it enables then?

Reply:
As explained [here](https://github.com/wazuh/wazuh-documentation/pull/6687#discussion_r1374858200), it enables or disabled the process to upload the rollover policy, but not the policy itself. Once the policy is uploaded, the only way to disable it is by removing it.

Reply:
Ok, Instead, we could invite the user to read the Index life management section so they can learn how to disable the policy by removing it.